src/App.tsx

```diff
@@ -1,36 +1,42 @@
 import { createTheme } from "@mui/material";
-import { ThemeProvider, StyledEngineProvider } from "@mui/system";
+import {
+	ThemeProvider as MuiThemeProvider,
+	StyledEngineProvider,
+} from "@mui/system";
 import React from "react";
 import "./App.css";
-import { LoginPage } from "./app/Login/Login";
+import { LoginPage } from "./pages/Login/LoginPage";
 import { Route, Switch } from "react-router-dom";
 import { Routes } from "./shared/Routes.constants";
 import { PrivateRoute } from "./shared/PrivateRoute";
 import Modules from "./pages/modules/ModulesView";
 import { ProblemEditorPage } from "./pages/ProblemEditor/ProblemEditorPage";
+import { ThemeProvider } from "@emotion/react";
```

Comment:
This tutorial recommended using two? https://dev.to/atonchev/material-ui-5-the-easiest-way-to-migrate-from-makestyles-to-emotion-1i9l

Reply:
to transition from make-styles to emotion

Reply:
Try to stick with what mui recommends, specially since it is a very recent version what we just migrated to. We are not trying anything special, so there is no need for roundabout solutions.

Reply:
ok. should be ready to resolve.